setup.py

```diff
@@ -49,7 +49,7 @@
             'flake8',
         ],
         coverage=[
-            'pytest-cov',
+            'coverage',
```

Comment:
I think the only reason to use pytest-cov is for distributed testing in which case we would also need xdist. Why use an additional layer, if we don't need it. The coverage syntax is also more declarative and easier to follow. 

For some more arguments see http://stackoverflow.com/a/16524426/5885568. 

I also want a consistent syntax in the docs and for tox. 

And actually at the moment most of our apps use coverage. 

BTW the decision was inspired by more.itsdangerous/ fd47bfc from @href.   ;-)